moveit_ros/hybrid_planning/global_planner/global_planner_plugins/src/moveit_planning_pipeline.cpp

```diff
@@ -67,6 +67,15 @@ bool MoveItPlanningPipeline::initialize(const rclcpp::Node::SharedPtr& node)
   node->declare_parameter<double>(PLAN_REQUEST_PARAM_NS + "max_acceleration_scaling_factor", 1.0);
   node->declare_parameter<std::string>("ompl.planning_plugin", "ompl_interface/OMPLPlanner");
 
+  // Planning Scene options
```

Comment:
These parameters were already present in `test/config/global_planner.yaml`, they just weren't being used anywhere.

Reply:
What if these were already declared?  should this check to see if the parameters exist before declaring them?

Should we give them defaults here that are not UNDEFINED?

Reply:
Can't you just use the PlanningSceneMonitorOptions? https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h#L57

Reply:
PlanningSceneMonitorOptions uses get_parameter_or(). Is that even valid without declaring the parameters first? Or is it going to use the default every time?

https://github.com/ros-planning/moveit2/blob/77e43f339cb6ee06679a424e21c72301797920d4/moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h#L62